import_data/osm_update.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+from os import path
+import subprocess
+import sys
+from datetime import datetime
+
+
+DESCRIPTION = "OpenStreetMap database update script"
+VERSION = "0.3/20191112"
+
+
+def exec_command(command):
+    proc = subprocess.Popen(command)
+    proc.wait()
+    return proc.poll()
+
+
+def get_time_now():
+    return datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+
+
+def log(msg):
+    print("[{}] {} :INFO: {}".format(get_time_now(), os.getpid(), msg))
+
+
+def log_error(msg):
+    print("[{}] {} :ERROR: {}".format(get_time_now(), os.getpid(), msg))
+
+
+def format_file_size(size):
+    if size > 1000000000:
+        return '{}GB'.format(size / 1000000000)
+    elif size > 1000000:
+        return '{}MB'.format(size / 1000000)
+    elif size > 1000:
+        return '{}KB'.format(size / 1000)
+    return '{}B'.format(size)
+
+
+def run_imposm_update(settings, entry):
+    imposm_config_file = path.join(settings["imposm_config_dir"], entry)
```

Comment:
(almost 100% off-topic)

I recently discovered that there is an higher-level module to handle paths in the standard library called [pathlib](https://docs.python.org/3.8/library/pathlib.html).

Maybe we should use it in future new projects, it seems slightly more convenient and maybe a bit safer? (Anyway, it would probably not be worth it to change anything in existing code)